README.md

```diff
@@ -1,180 +1,220 @@
-# activate virtualenv
 source ~/checker_env/bin/activate
 
-# get source code
-git clone https://github.com/Ericsson/codechecker.git
+# Check out CodeChecker
+git clone https://github.com/Ericsson/CodeChecker.git --depth 1
 cd codechecker
 
-# install required basic python modules
+# Install the basic Python modules needed for operation
 pip install -r .ci/basic_python_requirements
 
-# create codechecker package
+# Build and install a CodeChecker package
 ./build_package.py -o ~/codechecker_package
+
+# For ease of access, add the install directory for PATH
+export PATH="~/codechecker_package/CodeChecker/bin:$PATH"
+
 cd ..
-~~~~~~
+```
 
-## OS X
+### Mac OS X
 
-### Basic dependecy install & setup
-Tested on OS X El Capitan 10.11.5 & macOS Sierra 10.12 Beta
-~~~~~~{.sh}
+The following commands are used to bootstrap CodeChecker on OS X El Capitan
+10.11.5 and macOS Sierra 10.12 Beta.
 
-# on El Capitan System Integrity Protection (SIP) need to turn off
-- Click the  (Apple) menu.
-- Select Restart...
-- Hold down command-R to boot into the Recovery System.
-- Click the Utilities menu and select Terminal.
-- Type csrutil disable and press return.
-- Close the Terminal app.
-- Click the  (Apple) menu and select Restart....
+On El Capitan System Integrity Protection (SIP) needs to be turned off:
+  * Click the  (Apple) menu.
+  * Select Restart...
+  * Hold down command-R to boot into the Recovery System.
+  * Click the Utilities menu and select Terminal.
+  * Type csrutil disable and press return.
+  * Close the Terminal app.
+  * Click the  (Apple) menu and select Restart....
 
-# check out and build clang with extra tools
-How to: http://clang.llvm.org/get_started.html
+Check out and build LLVM/Clang with extra tools. Follow the [Get Started with
+LLVM/Clang](http://clang.llvm.org/get_started.html) documentation.
 
-# get dependencies
+```bash
+# Download and install dependencies
 brew update
 brew install doxygen thrift gcc git
 
-# get source code
+# Fetch source code
 git clone https://github.com/tmsblgh/codechecker-osx-migration.git
 cd codechecker
 
-# install required basic python modules
+# Install required basic python modules
 pip install -r .ci/basic_python_requirements
 
-# create codechecker package
+# Create codechecker package
 ./build_package.py -o ~/codechecker_package
 cd ..
-~~~~~~
+```
+
+Check your first project
+------------------------
 
-### Check a test project
-#### Check if clang or clang tidy is available
-The installed clang binary names should be ```clang``` or ```clang-3.6``` which depends on your Linux distribution.
-~~~~~~{.sh}
+### Testing and configuring LLVM/Clang availability
+
+_Clang_ and/or _Clang-Tidy_ must be available on your system before you can
+run analysis on a test project. The binaries are usually named `clang` or
+`clang-3.6` (and `clang-tidy` or `clang-tidy-3.6`, respectively), but this
+depends on your Linux distribution.
+
+```bash
 which clang-3.6
```

Comment:
Please update to he new clang and clang tidy versions here too clang-3.8, clang-tidy-3.8